src/toil/jobStores/googleJobStore.py

```diff
@@ -236,111 +176,104 @@ def writeFile(self, localFilePath, jobStoreID=None):
     @contextmanager
     def writeFileStream(self, jobStoreID=None):
         fileID = self._newID(isFile=True, jobStoreID=jobStoreID)
-        key = self._newKey(fileID)
-        with self._uploadStream(key, update=False) as writable:
-            yield writable, key.name
+        with self._uploadStream(fileID, update=True) as writable:
```

Comment:
Why did update change to True?

Reply:
I can't remember exactly, but I have a feeling tests will fail once I switch it back. Let's see what happens.